src/conf_mode/container.py

```diff
@@ -218,6 +220,10 @@ def verify(container):
             if v6_prefix > 1:
                 raise ConfigError(f'Only one IPv6 prefix can be defined for network "{network}"!')
 
+    if 'registry' in container:
+        for registry, registry_config in container['registry'].items():
+            if ('username' in registry_config) != ('password' in registry_config):
+                raise ConfigError(f'Must either not defined username and password, or defined both for registry {registry}')
```

Comment:
`If username or password is defined, so must be the other - check "{registry}" registry configuration!`